config/runtime.exs

```diff
@@ -239,3 +239,19 @@ if config_env() == :prod do
       redirect_uri: google_redirect_uri
   end
 end
+
+# OIDC Auth
+auth_oidc_env = System.get_env("AUTH_OIDC")
+
+if auth_oidc_env do
+  auth_oidc =
+    Jason.decode!(auth_oidc_env)
+    |> FzCommon.FzMap.map_to_keyword_list()
+    |> Enum.map(fn provider ->
+      {name, value} = provider
+      {name, put_in(value, [:redirect_uri], "#{external_url}/auth/oidc/#{name}/callback/")}
+    end)
```

Comment:
I think it's better to `Jason.decode(_, keys: :atoms)` then work with the maps directly.
You probably don't need to change your code other than removing the `map_to_keyword_list` call.